warehouse/models/mart/gtfs/fct_hourly_rt_feed_files_success.sql

```diff
@@ -0,0 +1,104 @@
+{{ config(materialized='incremental', unique_key = 'key') }}
+
+-- BigQuery does not do partition elimination when using a subquery: https://stackoverflow.com/questions/54135893/using-subquery-for-partitiontime-in-bigquery-does-not-limit-cost
+-- save max date in a variable instead so it can be referenced in incremental logic and still use partition elimination
+{% if is_incremental() %}
+    {% set dates = dbt_utils.get_column_values(table=this, column='dt', order_by = 'dt DESC', max_records = 1) %}
+    {% set max_date = dates[0] %}
+{% endif %}
+
+WITH
+
+int_gtfs_rt__daily_url_index AS (
+    SELECT *
+    FROM {{ ref('int_gtfs_rt__daily_url_index') }}
+),
+
+int_gtfs_rt__unioned_parse_outcomes AS (
+    SELECT *
+    FROM {{ ref('int_gtfs_rt__unioned_parse_outcomes') }}
+),
+
+int_transit_database__urls_to_gtfs_datasets AS (
+    SELECT *
+    FROM {{ ref('int_transit_database__urls_to_gtfs_datasets') }}
+),
+
+parse_outcomes AS (
+    SELECT *
+    FROM int_gtfs_rt__unioned_parse_outcomes
+    {% if is_incremental() %}
+    WHERE dt >= DATE '{{ max_date }}'
+    {% else %}
+        WHERE dt >= DATE_SUB(CURRENT_DATE(), INTERVAL 1 MONTH)
+    {% endif %}
+),
+
+hourly_totals AS (
+
+    SELECT
+
+        dt,
+        base64_url,
+        feed_type,
+        EXTRACT(HOUR FROM hour) AS download_hour,
+        SUM(
+            CASE parse_success WHEN TRUE THEN 1 ELSE 0 END
+        ) AS success_file_count_hour,
+        COUNT(*) AS file_count_hour
+
+    FROM parse_outcomes
+    GROUP BY
+        dt,
+        download_hour,
+        base64_url,
+        feed_type
+
+),
+
+pivot_hourly_totals AS (
+
+    SELECT *
+    FROM
+        (SELECT
+
+            dt,
+            download_hour,
+            base64_url,
+            feed_type,
+            (success_file_count_hour / file_count_hour) * 100 AS pct_success_hour
```

Comment:
my bad for not being clearer earlier but I do think it might be safer to leave this as just a decimal proportion in the warehouse and lean on Metabase for percentage formatting? so then would change the name to `prop_success_hour` instead of `pct_`